shared/profile/prove-enter-username-container.js

```diff
@@ -35,7 +35,7 @@ export default connector.connect(
     }
 
     return {
-      canContinue: profile.usernameValid,
+      canContinue: true,
```

Comment:
@MarcoPolo changed it to use state instead of going through redux so the validation needed to be moved into the component instead of reducer